driver-core/src/main/com/mongodb/internal/connection/CommandMessage.java

```diff
@@ -416,7 +408,7 @@ private static OpCode getOpCode(final MessageSettings settings, final ClusterCon
     }
 
     private static boolean isServerVersionKnown(final MessageSettings settings) {
-        return settings.getMaxWireVersion() >= FOUR_DOT_ZERO_WIRE_VERSION;
+        return settings.getMaxWireVersion() >= EARLIEST_WIRE_VERSION;
```

Comment:
I added this new constant so we don't keep needing to update this code every time the earliest supported version changes.

Reply:
Though perhaps this should just be `> 0`

Reply:
Yeah I think it can be as its validated in BaseCluster and this is only used to set the opcode. So can use `return settings.getMaxWireVersion() > MIN_WIRE_VERSION`.  Then `EARLIEST_WIRE_VERSION` isn't needed.